packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -155,6 +155,17 @@ export class RelationshipsController extends TransportController {
             throw this.newCacheEmptyError(RelationshipTemplate, template.id.toString());
         }
 
+        const queryForExistingRelationship = {
+            "peer.address": template.cache.createdBy.toString(),
+            status: { $in: [RelationshipStatus.Pending, RelationshipStatus.Active, RelationshipStatus.Terminated, RelationshipStatus.DeletionProposed] }
+        };
+
+        const existingRelationshipToPeer = await this.getRelationships(queryForExistingRelationship);
+
+        if (existingRelationshipToPeer.length !== 0) {
+            throw CoreErrors.relationships.relationshipAlreadyExists(existingRelationshipToPeer[0].status);
```

Comment:
is **already**Exists the correct naming for a deletion proposed relationship? I think the better wording would be **still**Exists in that case. But that's a detail, if you don't see it like that you can ignore this :)